<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegWatcherDialog</class>
 <widget class="QDialog" name="RegWatcherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Watch Address</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="leAddress">
     <property name="toolTip">
      <string>Hexadecimal address</string>
     </property>
     <property name="text">
      <string>0x</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QDialogButtonBox" name="btnBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblBank">
     <property name="text">
      <string>Bank</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cmbBank">
     <item>
      <property name="text">
       <string>A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmbType">
     <item>
      <property name="text">
       <string>Register</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Scratchpad RAM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Output port</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Input port</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnBox</sender>
   <signal>accepted()</signal>
   <receiver>RegWatcherDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBox</sender>
   <signal>rejected()</signal>
   <receiver>RegWatcherDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
